src/log_to_disk.c

```diff
@@ -61,12 +61,6 @@ void restart_band_timer(void) {
     }
 }
 
-/* score QSO and add to total points */
-void score_qso(void) {
```

Comment:
This has been moved to `score.c` as this single function required a bunch of dummies to be implemented in the test. It anyway fits better scoring than writing to disk.